Cubical/Experiments/Rng.agda

```diff
@@ -4,15 +4,16 @@
 -- rewrite has been postponed.
 
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
-module Cubical.Structures.Rng where
+module Cubical.Experiments.Rng where
```

Comment:
Oops, this wasn't caught by Travis... Can you add this file to Experiments.Everything?